packages/proxy/src/proxy.ts

```diff
@@ -187,11 +198,29 @@ export async function proxyV1({
   }
 
   // Caching is enabled by default, but let the user disable it
-  const useCacheMode = parseEnumHeader(
+  // XXX is there a clean way to specify "auto" caching behavior using only cache-control directives?
+  const legacyCacheMode = parseEnumHeader(
     CACHE_HEADER,
     CACHE_MODES,
     proxyHeaders[CACHE_HEADER],
   );
+
+  let useCacheMode = legacyCacheMode;
+  const cacheControlDirectives = cacheControlParse(
```

Comment:
should we also specifically intercept max-age=0, and take that to mean "don't cache"?